File: src/_/06/get_glyph_kind.ts
// SSoT: file:///Users/s0fractal/OMEGA/src/ontology/core/get_glyph_kind.md
import { KIND_CONTROL, KIND_CORE, GLYPH_TYPES } from "@g05";

export function get_glyph_kind(id: number): number {
  if (id <= 3) return KIND_CORE;
  if (id <= 15) return KIND_CONTROL;
  return id >> 3;
}

File: src/00/sigma_core/src/ontology_gen/06/get_glyph_kind.rs
// SSoT: src/ontology/core/get_glyph_kind.md
#![allow(unused_imports)]
use super::super::L05::*;

pub fn get_glyph_kind(id: u8) -> u8 {
    if id <= 3 {
      return KIND_CORE;
    }
    if id <= 15 {
      return KIND_CONTROL;
    }
    return id >> 3;
}
